docusaurus/docs/shared/_migrate-to-react-router-v6.mdx

```diff
@@ -63,3 +63,31 @@ return (
 #### 4. Follow the original `react-router` migration guide for more in-depth changes
 
 Visit the [official react-router v5 to v6 migration guide](https://reactrouter.com/en/main/upgrading/v5) for more information.
+
+#### 5. Fix test failures with location service methods
```

Comment:
It may make more sense to shift this to no.4 given the one above is more of a catch all to refer make to the react guide?